_config.yml

```diff
@@ -84,6 +84,17 @@ authors:
     twitter: aramonc
     gravatar: c41323a7dc18ea64a7f617e85ffb8c9a
     about: Besides Zumba<sup>&reg;</sup>, he is one of the co-organizers of the SoFloPHP users group, the SunshinePHP conference, and one of the technical leads for Code for Miami, a Code for America brigade.
+  ZumbaJustin:
+    name: Justin Oakley
+    email: justin.oakley@zumba.com
+    position: Software Engineer
+    start_date: 2015-03-30
+    github: ZumbaJustin
+    twitter: lordtakuban
+    gravatar: 8bf94254d96ded534cbb3a70540bfe84
+    about: Besides Zumba<sup>&reg;</sup>, he is working on various side projects.
+      Prior to the world of business application development, he worked on video games (<a href="imdb.com/name/nm2035116/">IMDB - Justin Oalkey</a>).
+      When not developing software, you may find him modifying one of his cars (<a href="mr2ner.com">MR2ner</a> | <a href="myhachiroku.com">My Hachi Roku</a>) or autocrossing.
```

Comment:
Thanks.  I thought they were not allowed in the markdown syntax based on the others in the file.  I will add them.